userguide/install_kernel_modules.rst

```diff
@@ -0,0 +1,71 @@
+Install Kernel Modules
+======================
+
+Install with DKMS
+^^^^^^^^^^^^^^^^^
+
+In order to support the mandatory kernel subsystems ashmem and binder for the
+Android container you have to install two
+`DKMS <https://en.wikipedia.org/wiki/Dynamic_Kernel_Module_Support>`_
+based kernel modules. The source for the kernel modules is maintained by the
+Anbox project `here <https://github.com/anbox/anbox-modules>`_.
+
+At the moment we only have packages prepared for Ubuntu in a PPA on Launchpad.
+If you want to help to get the packages in your favorite distribution please
+come and talk to us or submit a PR with the distribution specific packaging.
```

Comment:
Would be good if we add at the same time a page to tell people how to talk to the community/developers. Similar like what the godot project did: https://godot.readthedocs.io/en/stable/community/channels.html